### ISSUE TITLE
[UWP][Accessibility] Screen reader behavior with compact choice set

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#4423]
# Platform

What platform is your issue or question related to? (Delete other platforms).

* UWP

# Details

From Inclusive and Accessible Design Spec Review Report:

Using file Input.ChoiceSet.Compact.Label.json:

 - When drop down is expanded, screen reader should also announce that particular value is selected for both required and non-required fields.
- When drop down is collapse situation, screen reader only read type of widget, its label and if it is a required or non-required field. however, it does not read selected value of dropdown field.
